test/e2e-tests.js

```diff
@@ -889,6 +889,12 @@ describe('matrix-public-archive', () => {
       });
     });
 
+    describe('Room directory renders as expected', () => {
+      it('renders the room directory', async () => {
+        await getTestClientForHs(testMatrixServerUrl1);
+      });
+    });
```

Comment:
By "and a quick test", I just meant make sure it runs for you in Node.js 19.

Sorry this wasn't clear 🙇. We don't need this test. We already have many tests for making sure the archive and room directory works. We just don't run the test suite against Node.js 19 to catch it.

And I don't think we need to update the test matrix to also run with Node.js 19 yet. We can fix up the little ad-hoc things for now.

Reply:
> By ["and a quick test"](https://github.com/matrix-org/matrix-public-archive/issues/141#issuecomment-1319383479), I just meant make sure it runs for you with Node.js 19.

aha 😀

OK — as far as that goes, I can confirm that is runs as expected with Node.js v19.1.0

> Sorry this wasn't clear 🙇.

No worries — it gave me to a chance to get a test environment set up, and to look through the test-harness code, and to actually run the tests. So it was all win for me

Reply:
…and just now pushed a change that removes the test